tomcat/src/main/java/nextstep/jwp/util/FileNameUtil.java

```diff
@@ -0,0 +1,19 @@
+package nextstep.jwp.util;
+
+import java.util.Objects;
+
+public class FileNameUtil {
```

Comment:
```suggestion
public class FileUtils {
```

파일 관련 처리를 담당하는 유틸로 만들어주는게 어떨까요

Reply:
뭔가 사용이 한정적일거 같다라는 생각이 들었는데 너무 제한된 이름이었던 것 같네요 🤣